### PR TITLE
Backport: infra vms (wallaby)

### PR DESCRIPTION
**Pull request by markgoddard**
This change allows you to define additional VMs to deploy
on the seed-hypervisor.

Co-authored-by: Piotr Parczewski <piotr@stackhpc.com>
Co-authored-by: Will Szumski <will@stackhpc.com>
Co-authored-by: Mark Goddard <mark@stackhpc.com>
Story: 2008741
Task: 42095
Change-Id: I8055fc5eb0a9edadcb35767303c659922f2d07ca

(backported from currently unmerged upstream review, https://review.opendev.org/c/openstack/kayobe/+/783627)

### CODE REVIEW

File: ansible/dev-tools.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 - name: Ensure development tools are installed
-  hosts: seed-hypervisor:seed:overcloud
+  hosts: seed-hypervisor:seed:overcloud:infra-vms
```
Comment (cityofships) [inline ansible/dev-tools.yml:3]: Question: isn't hyphens usage discouraged in Ansible?

Comment (markgoddard) [reply ansible/dev-tools.yml:3]: Yes, but it's only a warning and we have other groups with hyphens in. I suppose we need to trade-off consistency vs future-proofing.

Comment (cityofships) [reply ansible/dev-tools.yml:3]: Indeed. It should probably be reported at Kolla Ansible level, if not already.

**Review by cityofships — APPROVED** [review]
Merged upstream already.

**Pull request merged**